packages/empty-state/src/example-graphics/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+
+import { Theme } from '@leafygreen-ui/lib';
+
+// @ts-ignore no type definition for SVG
```

Comment:
Should we use svgr?

Reply:
Would `svgr` allow us to get around TS issues? We've got SVGR set up on the repo, which the reason this kind of direct import works